python/maps_prototype_tas.ipynb

```diff
@@ -136,17 +137,17 @@
     "ax.set_title('EUR temperature difference (%) 50prec', fontsize=20)\n",
```

Comment:
Unit for temperature is degrees C or K, not %. Also typo in "50prec"

Reply:
Corrected!